pkg/cli/cli.go

```diff
@@ -102,6 +102,7 @@ const (
 	localBootloaderSrcFlag    = "local_bootloader_src"
 	localCVDHostPkgSrcFlag    = "local_cvd_host_pkg_src"
 	localImagesSrcsFlag       = "local_images_srcs"
+	localImagesZipSrcFlag     = "local_images_zip_src"
```

Comment:
is the user expected to create this zip file manually? or is it generated for them?

Reply:
good point, at the moment the flag value expects a single filename value, the file should have been already created.